core-bundle/src/Util/PackageUtil.php

```diff
@@ -12,7 +12,7 @@
 
 namespace Contao\CoreBundle\Util;
 
-use PackageVersions\Versions;
+use Composer\InstalledVersions;
 
 class PackageUtil
```

Comment:
shouldn't we deprecate our custom class as well and just use the Composer version?

Reply:
I would be fine with that as well, however, then we would have to duplicate the code of the `getContaoVersion()` method at several places.

Reply:
As it's a small static method, it could live somewhere else, couldn't it? Maybe in `ContaoCoreBundle`?

Reply:
Maybe simply `ContaoFramework::getVersion()`?

Reply:
I have deprecated the whole `PackageUtil` class in 6489dba43faed73f92d3e7589f45d28eb43cd7ac now.